ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/searchUtils.tsx

```diff
@@ -45,9 +45,10 @@ export function getOverviewCvesPath(workloadCvesSearch: WorkloadCvesSearch): str
 export function getEntityPagePath(
     workloadCveEntity: EntityTab,
     id: string,
+    vulnerabilityState: VulnerabilityState | undefined, // TODO Make this required when the ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL feature flag is removed
     queryOptions?: qs.ParsedQs
 ): string {
-    const queryString = getQueryString(queryOptions);
+    const queryString = getQueryString({ ...queryOptions, vulnerabilityState });
```

Comment:
If `vulnerabilityState` is `undefined`, will it be discarded or will it still be part of the query string?

Reply:
Under the hood this calls `qs.stringify`, which will omit `undefined` properties entirely.